hs_core/views/utils.py

```diff
@@ -544,7 +545,13 @@ def create_form(formclass, request):
 
 
 def get_my_resources_list(request):
```

Comment:
could you update this to take user instead of request? I don't like passing around request.

Reply:
Done.